src/components/TodoInput.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+function TodoInput(props) {
+  return (
+    <div className="form-group form-styled">
+      <h5 className="form-header">Add new item</h5>
+      <input name="name" type="text" placeholder="Name" value={props.name} onChange={props.handleChange} />
```

Comment:
state should be held as close to use case as possible.

See my comment above with validation. The same applies usually to the form values as well. If the state is used only per some component, add it to that component and dont make the root state too big that will contain everything.